python/ray/_private/usage/usage_constants.py

```diff
@@ -49,3 +49,11 @@
 EXTRA_USAGE_TAG_PREFIX = "extra_usage_tag_"
 
 USAGE_STATS_NAMESPACE = "usage_stats"
+
+KUBERNETES_SERVICE_HOST_ENV = "KUBERNETES_SERVICE_HOST"
+KUBERAY_ENV = "RAY_KUBERAY_IN_USE"
```

Comment:
normally usage stats related env vars are started with `RAY_USAGE_STATS_`

Reply:
Thanks for the pointer! Will modify accordingly.

Reply:
Fixed.